source/backend/api/Services/PropertyOperationService.cs

```diff
@@ -165,8 +164,22 @@ public IEnumerable<PimsPropertyOperation> ConsolidateProperty(IEnumerable<PimsPr
             return completedOperations;
         }
 
-        private static void CommonPropertyOperationValidation(IEnumerable<PimsPropertyOperation> operations)
+        private static void CommonPropertyOperationValidation(IEnumerable<PimsPropertyOperation> operations, IEnumerable<PimsProperty> dbSourceProperties)
         {
+
+            foreach (var sourceProperty in dbSourceProperties)
+            {
+                var operation = operations.FirstOrDefault(p => p.SourcePropertyId == sourceProperty.PropertyId);
+                if (operation == null)
+                {
+                    throw new BusinessRuleViolationException("All source properties must exist in the system.");
```

Comment:
All source properties must exist in the system.

Doesn't seem like an appropriate user facing error. Is this listed in the story somewhere?

If not I'd work with Ana or Julian for some text, this seems more like a developer appropriate error.

Reply:
Yep, not in the story. It is not really a error for the user, but in case the backend receives a bad request